src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+
+/**
+ * Overall Duke class encapsulating the duke bot, Chatty.
+ * @author Goh Ler Xuan
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Duke bot.
+     * @param filePath Path to data file with saved tasks from user's previous sessions

```

Comment:
I think there usually is a space between the params and the header of the comment right